py4cast/models.py

```diff
@@ -1,21 +1,27 @@
+"""
+Python module to plug mfai models into py4cast
+and also register any plugin found in the PYTHONPATH.
+"""
+
 import importlib
 import pkgutil
 from pathlib import Path
 from typing import Any, Tuple, Union
 
-from py4cast.models.base import ModelABC
-
-from .nlam.models import GraphLAM, HiLAM, HiLAMParallel
-from .vision.conv import HalfUnet, Unet
-from .vision.transformers import Segformer, SwinUNETR
-from .vision.unetrpp import UNETRPP
+from mfai.torch.models.base import ModelABC
```

Comment:
Is it possible to have a generic import rather than an enumeration ? Like importing all classes that in
herit from Model ABC ?

Reply:
We also could create the registry on the fly rather than manually

Reply:
Done